notebooks/teaching/README.md

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
@mcflugen can you help me understand why this .md was put here. I know that it means that the teaching folder will have this as a readme... but a bunch of this information is outdated AND I think we want to put most of this information on the readthedocs page rather than in the GitHub repo.... 

Is is just here so the content isn't lost?

Reply:
I just copied this over from the other repository without reviewing it. I agree that we don't need it.

Reply:
cool. I'll make sure to revise it when I do the doc revision related to tutorials that we discussed yesterday (and agreed should be a PR that happens after this one). 🎉